sky/setup_files/setup.py

```diff
@@ -109,11 +109,8 @@ def parse_readme(readme: str) -> str:
     # >=1.10.8 is needed for ray>=2.6. See
     # https://github.com/ray-project/ray/issues/35661
     'pydantic <2.0, >=1.10.8',
-    # Cython 3.0 release breaks PyYAML installed by aws-cli.
-    # https://github.com/yaml/pyyaml/issues/601
-    # https://github.com/aws/aws-cli/issues/8036
```

Comment:
We can probably leave this comment in if we update the lines below to disallow specific versions.

Reply:
Done!